### DISCUSSION TITLE
Hashrate reported by local miner is nearly double of what is reported by the pool

### DISCUSSION THREAD

**Discussion by StealthHydra179** [RainbowMiner/RainbowMiner#2880] [Q&A]
I have been testing rainbowminer recently, and I noticed that the hashrate reported by the local miner is:
![image](https://github.com/user-attachments/assets/9f1e98f6-0c46-4612-a9f0-e768431a2cdd)
while the pool only displays slightly above half that:
![image](https://github.com/user-attachments/assets/7a2dbf2e-00f2-46de-94d7-9602a4e1b18a)

Is there any possible reason for this?

**Comment by RainbowMiner** [comment]
Which miner is causing the trouble?

**Reply by StealthHydra179** [reply]
SrbMinerMulti on YespowerSUGAR

**Reply by RainbowMiner** [reply]
Ok, thanks. I will check it. How long have you been running the miner until you got that hashrate result on ZergPool?

**Reply by StealthHydra179** [reply]
Continuously for over 12 hours.

![image](https://github.com/user-attachments/assets/ab5cca53-38da-47b5-8305-09b1a6639fcc)
![image](https://github.com/user-attachments/assets/6fc93601-d9d2-494e-8116-c797eb8083d5)

Additionally, if I'm reading the numbers correctly, I think the estimation that RainbowMiner is making might be over-inflated in some way. For the same hashrate, the pool itself is reporting a lower profit.

**Reply by StealthHydra179** [reply]
I'm trying CpuminerPower right now, and it has only been ~20 minutes, but I've noticed that during this time the hashrate displayed pool-side is fluctuating a lot more, from about 100h/s to 500h/s, but still much lower than reported client-side.
![image](https://github.com/user-attachments/assets/2d07de1a-6122-46f7-b840-99bbc11485e7)
![image](https://github.com/user-attachments/assets/c6548b14-2855-420b-ae98-803106ef25b4)
![image](https://github.com/user-attachments/assets/98640383-934f-4f48-a8fe-439f1edfb21c)

**Reply by StealthHydra179** [reply]
@RainbowMiner Other miners have been fine so far, it has just been yespowerSugar. Is there any fix?

**Reply by RainbowMiner** [reply]
Strange that it hits yespowerSugar, only. No fix yet. I will run some tests.
